<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountingLSTreeGUI</class>
 <widget class="QWidget" name="AccountingLSTreeGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vista computo</string>
  </property>
  <layout class="QGridLayout" name="generalLayout">
   <item row="0" column="2">
    <widget class="QGroupBox" name="amountsGroupBox">
     <property name="title">
      <string>Importi</string>
     </property>
     <layout class="QGridLayout" name="amountsLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="totalAmountAccountedLabel">
        <property name="text">
         <string>Importo contabilizzato</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="totalAmountLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="totalAmountAccountedLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="totalAmountLabel">
        <property name="text">
         <string>Importo progetto</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="percentageAccountedLabel">
        <property name="text">
         <string>% contabilizzata</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="percentageAccountedLineEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTreeView" name="treeView">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="delPushButton">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addPushButton">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="printPushButton">
       <property name="text">
        <string>Stampa</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="priceGroupBox">
     <property name="title">
      <string>Prezzario</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QComboBox" name="priceListComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="currentPriceDataSetSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="priceListComboBoxLabel">
        <property name="text">
         <string>Prezzario</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="currentPriceDataSetSpinBoxLabel">
        <property name="text">
         <string>N. Preventivo</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
